angular/src/app/components/feature-metadata/feature-metadata.component.html

```diff
@@ -2,12 +2,14 @@
     <h6 class="">Metadata</h6>
 </div>
 <div class="grid-x grid-margin-x kv-table" *ngFor="let item of feature.properties | keyvalue">
-    <div class="cell small-4 text-left break">
-        <b>
-            <span> {{ item.key | titlecase }} </span></b
-        >
-    </div>
-    <div class="cell small-8 text-left break">
-        <code>{{ item.value | json }}</code>
-    </div>
+    <ng-container *ngIf="!item.key.startsWith('_hazmapper')">
+        <div class="cell small-4 text-left break">
+            <b
```

Comment:
This looks a bit off syntactically...

Reply:
hmm, predates this PR but in the diff as I'm moving it in a block to conditionally skip it if key is _hazmapper (that is where some non-user data is stored about the assets in rq).  fixing though as yes, very "off"